Jenkinsfile

```diff
@@ -37,64 +41,69 @@ pipeline {
                 sh 'poetry run ingest merge'
             }
         }
-        stage('neo-3-4-transform'){
+        // stage('neo4j-v3-transform'){
+        //     steps {
+        //         sh '''
+        //             docker rm -f neo || True
+        //             docker run --name neo -p7474:7474 -p7687:7687 -d --env NEO4J_AUTH=neo4j/admin neo4j:3.4.15
+        //             poetry run which kgx
+        //             ls -la
+        //             ls -la output/
+        //             poetry run kgx transform --stream --transform-config neo4j-transform.yaml > /dev/null
+        //             sleep 30s
+        //             mkdir neo4j-v3 || true
+        //             docker cp neo:/data ./neo4j-v3/data
+        //             cd neo4j-v3
+        //             tar czf neo4j-v3.tar.gz data
+        //             mv neo4j-v3.tar.gz ../output/
+        //         '''
+        //     }
+        // }
+        stage('neo4j-v4-transform'){
             steps {
                 sh '''
-                    docker rm -f neo || true
-                    docker run --name neo -p7474:7474 -p7687:7687 -d --env NEO4J_AUTH=neo4j/admin neo4j:3.4.15
-                    poetry run which kgx
-                    ls -la
-                    ls -la output/
-                    python3 -mvenv venv
-                    ./venv/bin/pip install kgx==1.5.2
-                    ./venv/bin/kgx transform --stream --transform-config neo4j-v3-transform.yaml
-                    sleep 30s
-                    mkdir neo4j-v3 || true
-                    docker cp neo:/data ./neo4j-v3/data
-                    cd neo4j-v3
-                    tar czf neo4j-v3.tar.gz data
-                    mv neo4j-v3.tar.gz ../output/
+                    docker rm -f neo || True
+                    mkdir neo4j-v4-data
+                    docker run -d --name neo -p7474:7474 -p7687:7687 -v neo4j-v4-data:/data --env NEO4J_AUTH=neo4j/admin neo4j:4.4
+                    poetry run kgx transform --stream --transform-config neo4j-transform.yaml
+                    docker stop neo
+                    docker run -v output:/backup -v neo4j-v4-data:/data --entrypoint neo4j-admin neo4j:4.4 dump --to /backup/monarch-kg.neo4j.dump
+                '''
+            }
+        }
+        stage('index') {
```

Comment:
The indexing stage should move to be after the uploading stage, so that it can list all of the newly updated files